repositories/repositories_test.go

```diff
@@ -23,10 +23,24 @@ type config struct {
 var EmployeeRepo EmployeeRepository
```

Comment:
All these vars can be grouped like:
var (
 EmployeeRepo EmployeeRepository
 ProductRepo ProductRepository
...
)